time pubTime

float64 effort
int32 jointNum
